lib/lti/variable_expander.rb

```diff
@@ -52,6 +52,8 @@ def self.default_name_expansions
     COURSE_GUARD = -> { @context.is_a? Course }
     TERM_START_DATE_GUARD = -> { @context.is_a?(Course) && @context.enrollment_term &&
                                  @context.enrollment_term.start_at }
+    TERM_NAME_GUARD = -> { @context.is_a?(Course) && @context.enrollment_term &&
```

Comment:
Maybe just use the safe navigation operator to simplify to `TERM_NAME_GUARD = -> { @context.is_a?(Course) && @context.enrollment_term&.name }`

Reply:
simplified (might want to consider also doing this to `TERM_START_DATE_GUARD` in the future)